node/cn/gasprice/gasprice.go

```diff
@@ -339,6 +331,33 @@ func (oracle *Oracle) getBlockValues(ctx context.Context, blockNum uint64, limit
 	}
 }
 
+func (oracle *Oracle) isRelaxedNetwork(header *types.Header) bool {
+	pset, err := oracle.gov.EffectiveParams(header.Number.Uint64() + 1)
+	if pset != nil && err == nil {
+		nextBaseFee := misc.NextMagmaBlockBaseFee(header, pset.ToKIP71Config())
+		return nextBaseFee.Cmp(big.NewInt(int64(pset.LowerBoundBaseFee()))) <= 0
+	}
+	return false
+}
+
+func (oracle *Oracle) checkCache(headHash common.Hash) (*big.Int, bool) {
```

Comment:
```suggestion
func (oracle *Oracle) readCacheChecked(headHash common.Hash) (*big.Int, bool) {
```

Reply:
Fixed in [8aef506](https://github.com/kaiachain/kaia/pull/44/commits/8aef506718cd74b9198d638dc9e75eec1eae4ffb)